connexion/lifecycle.py

```diff
@@ -24,7 +23,9 @@ def __init__(self,
 
     @property
     def json(self):
-        return self.json_getter()
+        if not hasattr(self, '_json'):
```

Comment:
I think this change is unrelated to your example, why did you add it?

Reply:
It is related. Actually it fixes the issue #1162. As opposed to FlaskApp, [this code](https://github.com/zalando/connexion/blob/master/connexion/apis/aiohttp_api.py#L318)  says that the body is recomputed each time we access `ConnexionRequest.json`. So if a validator changes the json, it is not reflected in the `body` argument because [here](https://github.com/zalando/connexion/blob/master/connexion/decorators/parameter.py#L90) it is recomputed again.

Reply:
If you don't believe, please try the example without this change and see that it behaves differently (the `foo` attribute is not added with the default value).